src/core/QHotkey/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
```

Comment:
Move to external

Reply:
I prefer to keep QHotKeys as it is, as library. It would be much easier to update it later. For now it doesn't support Wayland so I use it just for MacOS.
I don't see a problem in extra step for building library in link it staticaly.